apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.html

```diff
@@ -109,6 +109,42 @@
             </div>
         </div>
 
+        <div class="size-selection">
+            <div
+                class="small-box"
+                [ngStyle]="{
+                    'background-color': this.addToBasketForm.get('dimension').value === 'SMALL' ? '#AF7949' : '#FFFFFF'
+                }"
+            >
+                <span class="horizontal" *ngIf="getDesignType() === 'family-tree'">20 cm</span>
+                <span class="horizontal" *ngIf="getDesignType() === 'quotable'">15 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'family-tree'">20 cm</span>
+                <span class="vertical" *ngIf="getDesignType() === 'quotable'">15 cm</span>
+            </div>
+            <div
+                class="medium-box"
+                [ngStyle]="{
+                    'background-color': this.addToBasketForm.get('dimension').value === 'MEDIUM' ? '#AF7949' : '#FFFFFF'
```

Comment:
```suggestion
                    'background-color': this.addToBasketForm.get('dimension').value === 'MEDIUM' ? 'var(--light-second-action-color)' : '#FFFFFF'
```

Im not certain this will work with the var call in styling. If it doesn't then use this: #f3f3f3
It would be optimal with the var(--light-action-color); since it can be changed at a later time without having to rewrite all the hex colors on the pate.